docs/UserGuide.md

```diff
@@ -3,436 +3,713 @@ layout: page
+| `a/`     | `<ADDRESS>`           | The client's physical address                  | No specific restrictions                                                                        | ❌                     |
+| `j/`     | `<JOBNAME>`           | The client's job title or profession           | No specific restrictions                                                                        | ❌                     |
+| `i/`     | `<INCOME>`            | The client's annual income                     | Positive number or zero <br/> • Cannot include commas and decimal points<br/> • Must be numeric | ❌                     |
+| `t/`     | `<TIER>`              | The client's assigned tier level               | Predefined tiers:<br/> • Gold, Silver, Bronze, Reject                                           | ✔️                    |
+| `r/`     | `<REMARK>`            | General remarks about the client               | No specific restrictions                                                                        | ❌                     |
+| `ra/`    | `<REMARK TO APPEND>`  | Append information to the existing remark      | No specific restrictions                                                                        | ❌                     |
+| `rn/`    | `<NEW REMARK>`        | Replaces the existing remark with a new remark | No specific restrictions                                                                        | ❌                     |    
 
-### Feature 3: Remove Old Customer {#remove-old-customer}
+> 💡 **Pro Tip:**
+> 
+> Ensure every flag is followed by a valid argument!
+>
+> Providing a flag without an accompanying argument will result in an error and prevent the command from executing properly.
 
-**Purpose:**  
-This feature allows you to remove records of customers who are no longer using your credit card services.
+## 4.5 Using Commands
+To get started, simply type a command into the command box and hit **Enter**.
+
+Some initial commands to try:  
+**Viewing All Clients**
+* `list`: This command displays all clients currently in your database, making it easy to browse through entries.
+
+**Adding a New Client**
+* `add n/Jane Doe p/87654321 e/jane@example.com a/123 Jane Road j/doctor i/120000`: Adds Jane Doe to your database with detailed contact information, job title, and income.
+
+**Editing a Client's Information**
+* `edit 1 p/12345678`: Updates the phone number of the first client in your list to `12345678`.
+* `edit 4 rn/Updated remarks here`: Replaces the remarks of the fourth client with "Updated remarks here".
+
+**Removing a Client**
+* `delete 3`: Removes the third client from your list. Ensure you have the correct index to avoid deleting the wrong client. 
+
+**Searching for a Client**
+* `filter n/Jane`: Finds all client named Jane in your database. It’s a powerful tool for quickly locating clients or filtering for a specific type of client.
```

Comment:
Should be `Finds all clients named Jane`.

Reply:
Nice catch! Fixed.